### ISSUE TITLE
Book 05: Sentence 046

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#23]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/5311d944-0718-47c3-b1f5-93e436057ad5)

```
# sent_id = Book_05_-_The_Omens.txt:46
# text = No ioan, intla ie aca mococoa: in jpan tzatzitiqujca, qujtoaia ca aiocmo qujcaz: ca ticcaoazque.
# text[norm] = No ihuan, intla ye aca mococoa: in ipan tzatzitiquiza, quihtoaya ca ayocmo quizaz: ca ticcahuazque.
# text[orig] = No·ioan·,·intla·ie·aca·mococoa·:·inj¶pan·tzatzitiqujca·,·qujtoaia·ca·aioc¶mo·qujcaz·:·caticcaoazque·.
# text[osp] = [...] estaua algun enfermo: luego le pronosticauan la muerte.
# text[spa] = También con eso, si ya alguien se enferma: que en él se pone a chillar, decían que ya no saldrá, que lo dejaremos.
# references = A&D/169:3
1	No	no	ADV	_		13	advmod	_	Folio=7v|Paragraph=3|Line=13|Norm=No|Gloss=también|Analysed=Yes
2	ioan	huan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	13	obl	_	Folio=7v|Paragraph=3|Line=13|Norm=ihuan|Gloss=con|Analysed=Yes
3	,	,	PUNCT	_		7	punct	_	Folio=7v|Paragraph=3|Line=13|Norm=,|Analysed=Yes
4	intla	tla	PRON	_	_	7	mark	_	Folio=7v|Paragraph=3|Line=13|Norm=intla|Gloss=algo
5	ie	ye	ADV	_	_	7	advmod	_	Folio=7v|Paragraph=3|Line=13|Norm=ye|AmbigNorm=True|Override=False
6	aca	aca	DET|PRON	_	_	7	nsubj	_	Folio=7v|Paragraph=3|Line=13|Norm=aca
7	mococoa	cocoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	13	advcl	_	Folio=7v|Paragraph=3|Line=13|Norm=mococoa|Analysed=Yes
8	:	:	PUNCT	_		11	punct	_	Folio=7v|Paragraph=3|Line=13|Norm=:|Analysed=Yes
9	in	in	DET|SCONJ	_	_	11	_	_	Orig=inj¶·pan|Folio=7v,7v|Paragraph=3,3|Line=13,14|Norm=in
10	jpan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	11	obl	_	Orig=inj¶·pan|Folio=7v,7v|Paragraph=3,3|Line=13,14|Norm=ipan|Analysed=Yes
11	tzatzitiqujca	tzatzi	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr	7	_	_	Folio=7v|Paragraph=3|Line=14|Norm=tzatzitiquiza|Analysed=Yes
12	,	,	PUNCT	_		13	punct	_	Folio=7v|Paragraph=3|Line=14|Norm=,|Analysed=Yes
13	qujtoaia	ihtoa	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	0	root	_	Folio=7v|Paragraph=3|Line=14|Norm=quihtoaya|Analysed=Yes
14	ca	cah|ca	VERB	_	_	13	_	_	Folio=7v|Paragraph=3|Line=14|Norm=ca|AmbigNorm=True|Override=False
15	aiocmo	ayocmo	ADV	_		16	advmod	_	Orig=aioc¶·mo|Folio=7v,7v|Paragraph=3,3|Line=14,15|Norm=ayocmo|Gloss=ya.no|Analysed=Yes
16	qujcaz	quiza	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Fut|VerbForm=Fin	14	_	_	Folio=7v|Paragraph=3|Line=15|Norm=quizaz|Analysed=Yes
17	:	:	PUNCT	_		18	punct	_	Folio=7v|Paragraph=3|Line=15|Norm=:|Analysed=Yes
18	ca	cah|ca	VERB	_	_	13	_	_	Orig=caticcaoazque|Folio=7v|Paragraph=3|Line=15|Norm=ca|AmbigNorm=True|Override=False
19	ticcaoazque	cahua	VERB	_	Number[obj]=Sing|Number[subj]=Plur|Person[obj]=3|Person[subj]=1|Subcat=Tran|Tense=Fut|VerbForm=Fin	18	__	Orig=caticcaoazque|Folio=7v|Paragraph=3|Line=15|Norm=ticcahuazque|Analysed=Yes
20	.	.	PUNCT	_		13	punct	_	Folio=7v|Paragraph=3|Line=15|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
Questions:
- Status of *ca*, is this `VERB` or `PART`. I suspect `PART`...
- Are *intla ye aca mococoa* and  *in ipan tzatzitiquiza* two separate `advcl` of *quihtoaya?
- For *no ihuan* see #5.

**Comment by ftyers** [comment]
López Austin:
> Y también si alguno enfermaba y sobre él se ponía a cantar, de­cían: "Ya no saldrá; ya lo entregaremos."

Note that this is somewhat like #36 in terms of the conjunction semantics.

**Comment by ftyers** [comment]
Here is an option:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/5ee10b70-0f3a-4e0a-bba0-7223a08cac71)

But we should make sure to treat similar examples of *intla X: in Y, Z* the same.

**Comment by Lguyogiro** [comment]
This looks right to me. And yeah, we should make sure to treat these constructions similarly (I think there are a number of them), at least the ones that have "quihtoaya" as the Z verb.

**Comment by ftyers** [comment]
Merged in 01314fc.
